homeassistant/components/vera/config_flow.py

```diff
@@ -0,0 +1,25 @@
+"""Config flow for Vera."""
+from requests.exceptions import RequestException
+
+from homeassistant import config_entries
+
+from .const import CONF_CONTROLLER, DOMAIN
+
+
+class VeraFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Vera config flow."""
+
+    async def async_step_import(self, config: dict):
+        """Handle a flow initialized by import."""
+        base_url = config.get(CONF_CONTROLLER)
+
+        controller = self.hass.data[DOMAIN].controller
```

Comment:
Create a new controller instead. When the user initiates a config flow, ie step user, the component will not be set up yet at this point. So we shouldn't depend on hass.data stored things in the config flow.